code/index.html

```diff
@@ -2,21 +2,59 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+   <body>
+     <!-- Some kind of logo and navbar -->
+    <section class="header">
+      <div class="logo">
+        <h1>wood</h1>
+        <p>magazine</p>
+      </div>
+      <div class="navbar">
+        <ul>
+        <li><a href="https://www.wwf.se">About</a></li>
+        <li><a href="https://www.wwf.se">Archive</a></li>
+        <li><a href="https://www.wwf.se">Gallery</a></li>
+        </ul>
+      </div> 
     </section>
-
-    <section class="">
-      <!-- The other news  -->
+    <!-- Headnews and other news -->
+    <section class="news">
+      <div class="headnews">
+        <img src="forest.jpeg">
+        <h2>//NOW <br>
+        Sustainable forestry?</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+         Etiam et neque sem.</p> 
+        <p>Arcu facilisis at risus non, 
+         auctor lacinia metus. Nulla ut leo est. Curabitur posuere 
+         facilisis tincidunt. Nulla facilisi. </p>
+      </div>
+      <div class="news2">
```

Comment:
Just a thought here- do you need different classes fo the different newsitems (news2-news5) or could they actually be the same class? Cause if so it would probably save you some code in the CSS:)